assets/styles/main.css

```diff
@@ -0,0 +1,229 @@
+html{
+    background-color:  #ECEEF1;
+    font-family: 'Roboto', sans-serif;
+}
+
+body{
+    background-color: #ECEEF1;
+}
+
+/******************** HEADER ***************/
+.flex-container{
+    display: flex;
+}
+
+header{
+    /* background-color: pink; */
+    height: 10vh;
+    min-height: 50px;
+    width: 100vw;
+}
+
+.header-flex{
+    justify-content: flex-end;
+    align-items: center;
+}
+
+.header-flex > div{
+    padding: 10px;
+    text-align: end;
+}
+
+.sign-in-button{
+    font-size: 16px;
+    background-color: #0077C5;    
+    color: white;
+    border-radius: 3px;
+    border: 1px;
+    border-color: black;
+    padding: 12px;
+    height: 40px;
+    text-align: center;
+    vertical-align: text-bottom;
+    
+}
+
+.lock-icon{
+    font-size: 14px;
+}
+/********************************************/
+
+/******************** MAIN ***************/
+main{
+    /* background-color: yellow; */
+    height: fit-content;
+    min-height: 650px;
+    width: 100vw;
+}
+
+.main-flex{
+    justify-content: center;
+    align-content: center;
+}
+
+/******************** MAIN -> FORM ***************/
+.form-container{
+    background-color: white;    
+    margin-top: 10vh;     
+    width: 450px; /*30%*/
+    /* max-width: 450px;  */
```

Comment:
Except for comments above different sections like the "main form" comment above, always leave your CSS free of commented unneeded styles.